GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java

```diff
@@ -282,6 +288,13 @@ public GVRTexture loadEmbeddedTexture(TextureRequest request, AiTexture aitex, G
         public void onModelLoaded(GVRContext context, GVRSceneObject model, String modelFile) {
             mModel = model;
             Log.d(TAG, "ASSET: successfully loaded model %s %d", modelFile, mNumTextures);
+            if (mUserHandler != null)
+            {
+                mUserHandler.onModelLoaded(context, model, modelFile);
```

Comment:
You might want to wrap direct calls into user's code with try-catch to avoid their exceptions breaking the fw.